packages/components/src/CollapsiblePanel/CollapsiblePanel.scss

```diff
@@ -3,14 +3,18 @@ $tc-collapsible-panel-padding-normal: $padding-normal !default;
 $tc-collapsible-panel-padding-larger: $padding-larger !default;
 $tc-collapsible-panel-btn-color: #555964;
 $tc-collapsible-selectable-panel-content-description-color: #918F8F !default;
-$tc-collapsible-selectable-panel-title-color: #63AEBD !default;
-$tc-collapsible-selected-panel-border-color: #6AA9D1 !default;
+$tc-collapsible-selected-panel-color: #4F93A7 !default;
 
 @mixin type-col($width: 100%) {
 	flex-basis: $width;
-	white-space: nowrap;
-	text-overflow: ellipsis;
+	display: flex;
 	overflow: hidden;
+
+	> * {
```

Comment:
Because of this piece of code buttons on hover are cut
![peek 13-04-2017 17-13](https://cloud.githubusercontent.com/assets/18534166/25011129/9cc809da-206c-11e7-873b-c2f5d1838a39.gif)